security/middleware.py

```diff
@@ -247,6 +247,60 @@ def process_response(self, request, response):
         return response
 
 
+class ClearSiteDataMiddleware(BaseMiddleware):
+    """
+    Sends Clear-Site-Data HTTP response header on requests that match
+    CLEAR_SITE_DATA_URL_WHITELIST.
+
+    Clears browsing data (cookies, storage, cache) associated with the
+    requesting website. Allows web developers to have more control over the
+    data stored locally by a browser for their origins.
+
+    Reference:
+
+    - Clear-Site-Data: "cache", "cookies", "storage", "executionContexts", "*"
+      <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data>`_
+    """
+
+    REQUIRED_SETTINGS = ('CLEAR_SITE_DATA_URL_WHITELIST',)
+    OPTIONAL_SETTINGS = ('CLEAR_SITE_DATA_HEADERS')
```

Comment:
I think calling the values "headers" might be a little confusing because clear site data is a header itself. We could call it directives like [mozilla](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data)

Reply:
Done